### PR TITLE
Saving DICOM from .vdx files which can be loaded into Eclipse

### LINKED ISSUE #188: DICOM export for .vdx files compatible with Eclipse
According to Alina SOPInstanceUID tag is missing: 

> At first sight, something important that is missing is the referenced SOPInstanceUID. In the case of structure sets the Reference SOP Instance UID links the contours in the structure set to the specific slice in the dataset, this tag needs to be added, for each contour and each slice.

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #188

### CODE REVIEW

**Comment by grzanka** [comment]
@nbassler ready to be squashed and merged. There will be more issues, but I've the confirmation from IFJ that contours can be loaded in Eclipse.

File: pytrip/ctx.py [hunk 1]
```diff
@@ -85,8 +85,9 @@ def create_dicom(self):
             ds.RescaleSlope = 1.0
             ds.PixelRepresentation = 1
             ds.ImagePositionPatient = ["%.3f" % (self.xoffset * self.pixel_size),
-                                       "%.3f" % (self.yoffset * self.pixel_size), "%.3f" % (self.slice_pos[i])]
-            ds.SOPClassUID = '1.2.840.10008.5.1.4.1.1.2'
+                                       "%.3f" % (self.yoffset * self.pixel_size),
+                                       "%.3f" % (self.slice_pos[i])]
+            ds.SOPClassUID = '1.2.840.10008.5.1.4.1.1.2'  # CT Image Storage SOP Class
```
Comment (nbassler) [inline pytrip/ctx.py]: Do we know anything about all those magic numbers?

File: pytrip/utils/trip2dicom.py [hunk 2]
```diff
@@ -59,7 +59,7 @@ def main(args=sys.argv[1:]):
     ctx_path = ctx_basename + ".vdx"
     if os.path.exists(ctx_path):
         print("Convert structures")
-        v = pt.VdxCube(c)
+        v = pt.VdxCube(content="", cube=c)
```
Comment (nbassler) [inline pytrip/utils/trip2dicom.py]: what is "content" ?

File: pytrip/vdx.py [hunk 3]
```diff
@@ -933,9 +962,7 @@ def add_dicom_contour(self, dcm):
         offset.append(float(self.cube.yoffset))
         offset.append(float(min(self.cube.slice_pos)))
         self.contour.append(
-            Contour(pytrip.res.point.array_to_point_array(
-                np.array(
-                    dcm.ContourData, dtype=float), offset)))
+            Contour(pytrip.res.point.array_to_point_array(np.array(dcm.ContourData, dtype=float), offset)))
```
Comment (nbassler) [inline pytrip/vdx.py]: if you have figured out what array_to_point_array() is doing, can you update the docstring?

File: pytrip/ctx.py (hunk 1 above)
Comment (grzanka) [inline pytrip/ctx.py]: Yes, this is a reason why I've added a comment.
Number '1.2.840.10008.5.1.4.1.1.2' corresponds to `CT Image Storage SOP Class` in DICOM terminology.

File: pytrip/utils/trip2dicom.py (hunk 2 above)
Comment (grzanka) [inline pytrip/utils/trip2dicom.py]: See issue https://github.com/pytrip/pytrip/issues/190

File: pytrip/vdx.py (hunk 3 above)
Comment (grzanka) [inline pytrip/vdx.py]: I haven't figured out. This change here was just a refactoring

File: pytrip/utils/trip2dicom.py (hunk 2 above)
Comment (nbassler) [inline pytrip/utils/trip2dicom.py]: ahaa

**Pull request merged**